powerstore/datasource_protection_policy_test.go

```diff
@@ -0,0 +1,86 @@
+package powerstore
+
```

Comment:
Please add negative scenarios as well

Reply:
Also you can add the test cases with empty id or empty name

Reply:
done